Frontend/Gmfctn/src/app/modules/dashboard/badges/badges.component.ts

```diff
@@ -0,0 +1,9 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
Since you do not use OnInit in this file, you can remove it from the imports
```suggestion
import { Component } from '@angular/core';
```

Reply:
Fixed.